1_prep.R

```diff
@@ -7,17 +7,18 @@ p1 <- list(
   # TODO - transfer to Denali using globus
   # list of lake-specific attributes for nml modification
   tar_target(p1_nml_list_rds, '1_prep/in/nml_list.rds', format = 'file'),
-  tar_target(p1_nml_lake_ids, names(readr::read_rds(p1_nml_list_rds))),
-  # lake - GCM cell crosswalk, filtered to only those lakes that are
-  # in nml list (able to be modeled by GLM)
-  tar_target(p1_lake_cell_xwalk_csv, '1_prep/in/lake_cell_xwalk.csv', format = 'file'),
-  tar_target(p1_lake_cell_xwalk_df, 
-             readr::read_csv(p1_lake_cell_xwalk_csv, col_types=cols()) %>%
-               filter(site_id %in% p1_nml_lake_ids)),
+  tar_target(p1_nml_site_ids, names(readr::read_rds(p1_nml_list_rds))),
+  # lake - GCM cell - GCM tile crosswalk, filtered to only those lakes
+  #  that are in nml list (able to be modeled by GLM)
+  tar_target(p1_lake_cell_tile_xwalk_csv, '1_prep/in/lake_cell_tile_xwalk.csv', format = 'file'),
+  tar_target(p1_lake_cell_tile_xwalk_df, 
+             readr::read_csv(p1_lake_cell_tile_xwalk_csv, col_types=cols()) %>%
+               filter(site_id %in% p1_nml_site_ids) %>%
+               arrange(site_id)),
 
   # Define mapping variables
-  tar_target(p1_lake_ids, pull(p1_lake_cell_xwalk_df, site_id)),
-  tar_target(p1_cell_nos, unique(p1_lake_cell_xwalk_df$cell_no)),
+  tar_target(p1_site_ids, pull(p1_lake_cell_tile_xwalk_df, site_id)),
+  tar_target(p1_cell_nos, unique(p1_lake_cell_tile_xwalk_df$cell_no)),
```

Comment:
Just a note on style consistency - lines 20 & 21 use two different ways to extract a column as a vector. One uses `$` and one uses `pull()`. Both work, but would be nice to use the same approach, especially for targets operating on the same data right next to each other like this. IMO `pull()` is usually most beneficial when you have a string of tidyverse commands `%>%` and you want to keep that flow going but need to get a single column at the end. If it is a standalone step to pull a column out, I tend to lean toward `$`.